pkg/api/handlers/compat/info.go

```diff
@@ -181,6 +182,13 @@ func getSecOpts(sysInfo *sysinfo.SysInfo) []string {
 		// FIXME: get profile name...
 		secOpts = append(secOpts, fmt.Sprintf("name=seccomp,profile=%s", "default"))
 	}
+	if rootless.IsRootless() {
+		secOpts = append(secOpts, "name=rootless")
+	}
+	if selinux.GetEnabled() {
+		secOpts = append(secOpts, "name=selinux")
```

Comment:
These look good to me, although I don't know what the docker protocol expects?

Should we also return info for apparmor?

Reply:
Podman already reports apparmor. This[^1] is the equivalent code from moby, they also report `userns` and `cgroupns`.

[^1]: https://github.com/moby/moby/blob/5bf405b2afea6daf3960ad91583b0d290afd843f/daemon/info.go#L150-L175

Reply:
Awesome, LGTM